geoportailv3/static/js/query/default.html

```diff
@@ -0,0 +1,9 @@
+<h1>{{layers['layerLabel'] | translate}}</h1>
```

Comment:
Any reason for using the bracket notation here? We always use the dot notation in templates.